src/main/resources/application-test.yaml

```diff
@@ -1,45 +1,16 @@
-application:
-  group: zagrodnicy (Bycza-zagroda)
-  version: 0.5.0
-
 #Datasource
 spring:
-  banner:
-    location: classpath:banner.txt
   datasource:
     url: jdbc:mysql://localhost:3306/trackexpensesapptest?useUnicode=true&serverTimezone=UTC
     username: root
     password: root
-    driver-class-name: com.mysql.cj.jdbc.Driver
-    hikari:
-      connection-timeout: 500
-      maximum-pool-size: 5
 
```

Comment:
Why did you delete pool-size and timeout from test profile?

Reply:
This setting was added to fix an issue with test containers that no longer exists after recent changes to the testcontainers configuration.